doc/rados/operations/monitoring-osd-pg.rst

```diff
@@ -255,112 +253,6 @@ To query a particular placement group, execute the following::
 	
 Ceph will output the query in JSON format.
```

Comment:
So does the output of `ceph pg {poolnum}.{pg-id} query` always return JSON? Just wondering, as the pg dump example above explicitly  suggests adding `--format=json` as a parameter.

Reply:
Yes the `pg query` shows always json. But the `pg dump` output can be in either format.